service/src/main/resources/application.yml

```diff
@@ -26,6 +26,13 @@ server:
 # Spring
 ###############################################################################
 spring:
+  mail:
+    host: ''
+    username: ''
+    password: ''
+    port: 465
+    protocol: 'smtps'
+    debug: false
```

Comment:
I used my personal mail when working on this issue. So this props must be configurated somehow

Reply:
@jmprathab 
We should have some SMTP for our service.
This can be configured via environment properties to not store that in the repository.
Do you have some SMTP configured already?